src/lsptoolshost/projectRestore/restore.ts

```diff
@@ -18,32 +18,35 @@ import { getCSharpDevKit } from '../../utils/getCSharpDevKit';
 
 let _restoreInProgress = false;
 
-export function registerRestoreCommands(context: vscode.ExtensionContext, languageServer: RoslynLanguageServer) {
-    const restoreChannel = vscode.window.createOutputChannel(vscode.l10n.t('.NET NuGet Restore'));
+export function registerRestoreCommands(
+    context: vscode.ExtensionContext,
+    languageServer: RoslynLanguageServer,
+    csharpOutputChannel: vscode.LogOutputChannel
+) {
     context.subscriptions.push(
         vscode.commands.registerCommand('dotnet.restore.project', async (_request): Promise<void> => {
             if (getCSharpDevKit()) {
                 appendLineWithTimestamp(
-                    restoreChannel,
+                    csharpOutputChannel,
                     "Not handling command 'dotnet.restore.project' from C# extension, because C# Dev Kit is expected to handle it."
```

Comment:
Should we preface these strings, `[.NET Restore]`, now that they will be mixed into the C# log?

Reply:
That seems fine, though, I am wondering if there is a way to make a derived channel that bakes that in just like the timestamps seem to be be getting baked in.